src/app/screens/BookDetail/components/BookDetail/styles.js

```diff
@@ -0,0 +1,72 @@
+import { StyleSheet } from 'react-native';
+
+import { white, black, darkGray, gray, skyBlue } from '../../../../../constants/colors';
+import * as FontSize from '../../../../../constants/fontSizes';
+import * as Dimensions from '../../../../../constants/dimensions';
+import * as Margins from '../../../../../constants/margins';
+import * as Paddings from '../../../../../constants/paddings';
```

Comment:
merge margins and padding into dimensions and use appropriate names. Also import only what you need like in colors.